cypress-io/cypress
# feat: implement the ui for migration to 10.0

Closes [UNIFY-514](https://cypress-io.atlassian.net/browse/UNIFY-514)

### ScreenShots

These screenshots have been taken with the `cy.screenshot()` feature

![MigrationWizard -- renders Automatic rename](https://user-images.githubusercontent.com/5592465/146209266-e250b117-686a-464c-9e05-5f8905988f6c.png)

![MigrationWizard -- renders Config File migration](https://user-images.githubusercontent.com/5592465/146209271-1d67f26d-a7be-4071-a38f-da11570e2905.png)

![MigrationWizard -- renders Manual rename](https://user-images.githubusercontent.com/5592465/146209275-1a24a681-eee8-4a4b-95f1-eefb2ce9991b.png)

### How to test

- Start component testing in the launchpad
- Move to the migration folder
- See the tests passing

### Todo next

in a separate PR, we should wire up the data to
- Detect that a migration is needed and suggest it (not force it)
- List the files detected by the original globs + compute the modified list
- Create mutation to actually apply Rename spec files

- Show contents of `cypress.json` & `cypress/plugins/index.js` and what it will look like when merged
- Make the mutation to apply the change: plugins + `cypress.json` -> `cypress.config.js`
